src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
 public class ColorSupplier {
+    private final int randomColorIndex = (int) (Math.random() * Color.values().length);
+
     public String getRandomColor() {
```

Comment:
it's not actually random value in your solution - it'll be calculated once and every time we'd get identic color
```suggestion
    private final Random random = new Random();

    public String getRandomColor() {
        int index = random.nextInt(Color.values().length);
        return Color.values()[index].name();
    }
```

Reply:
updated with your suggestion.

but why the next structure is not correct (from my very firs PR)? 
If I have the only one method in class why I need to create class level variable?

public String getRandomColor() {
        int randomColorIndex = new Random().nextInt(Colors.values().length);
        return Colors.values()[randomColorIndex].name();
        }